docs/src/creating_custom_plots.qmd

```diff
@@ -1,84 +1,57 @@
-### A Pluto.jl notebook ###
-# v0.19.16
-
-using Markdown
-using InteractiveUtils
-
-# ╔═╡ e5e15f39-027c-42aa-a765-588ec1cd0b63
-using Pkg, InteractiveUtils
-
-# ╔═╡ b1784f52-0cf2-11ed-32ba-69a8f63b48a9
-# Remove this cell to use release versions of dependencies 
-# hideall
-let
-    docs_dir = dirname(@__DIR__)
-    pkg_dir = dirname(docs_dir)
-
-    Pkg.activate(docs_dir)
-    Pkg.develop(; path=pkg_dir)
-    Pkg.instantiate()
-end;
-
-# ╔═╡ 34ee7907-49e7-4e32-9bd5-1c5d490a13d8
-begin
-    using ArviZ, DimensionalData, DataFrames, Statistics, AlgebraOfGraphics, CairoMakie
-    using AlgebraOfGraphics: density
-    set_aog_theme!()
-end;
-
-# ╔═╡ 41b1ddca-20dc-4821-9cec-d7192ef34795
-using PlutoUI
-
-# ╔═╡ 70f66296-349c-48a5-9b83-cc5c6cdbd514
-md"""
-# Creating custom plots
-"""
-
-# ╔═╡ 47c7a1fd-0862-49d5-879d-d59d321ce014
-md"""
-While ArviZ includes many plotting functions for visualizing the data stored in `InferenceData` objects, you will often need to construct custom plots, or you may want to tweak some of our plots in your favorite plotting package.
+---
+title: "Creating custom plots"
+---
+
+While ArviZ includes many plotting functions for visualizing the data stored in [`InferenceData`](@ref) objects, you will often need to construct custom plots, or you may want to tweak some of our plots in your favorite plotting package.
 
 In this tutorial, we will show you a few useful techniques you can use to construct these plots using Julia's plotting packages.
-For demonstration purposes, we'll use Makie.jl and AlgebraOfGraphics.jl, which can consume `Dataset` objects since they implement the Tables interface.
+For demonstration purposes, we'll use Makie.jl and AlgebraOfGraphics.jl, which can consume [`Dataset`](@ref) objects since they implement the Tables interface.
 However, we could just as easily have used StatsPlots.jl.
-"""
 
-# ╔═╡ 7bdff7ba-408c-4893-9857-7cf328301a34
-md"""
+```{julia}
+#| echo: false
+#| code-fold: true
+#| output: false
+using Pkg;
+cd(@__DIR__)
+Pkg.activate(".."); # for reproducibility use the docs environment.
+```
+
+```{julia}
+using ArviZ, DimensionalData, DataFrames, Statistics, AlgebraOfGraphics, CairoMakie
+using AlgebraOfGraphics: density
+set_aog_theme!()
+```
+
 We'll start by loading some draws from an implementation of the non-centered parameterization of the 8 schools model.
 In this parameterization, the model has some sampling issues.
-"""
 
-# ╔═╡ cbd81558-e355-4e7a-ba51-0ba9299cb558
+```{julia}
 idata = load_example_data("centered_eight")
+```
 
-# ╔═╡ 898d83d1-cd3a-47a4-84b1-f2cf0f7bf959
+```{julia}
 idata.posterior
+```
 
-# ╔═╡ da4e989e-a08c-45af-8e55-20ac3ed58745
-md"""
 The plotting functions we'll be using interact with a tabular view of a `Dataset`.
 Let's see what that view looks like for a `Dataset`:
-"""
 
-# ╔═╡ 52ef5ef5-2ea8-4e1a-8cd5-ba3aa5da6d9b
+```{julia}
 df = DataFrame(idata.posterior)
+```
```

Comment:
Currently this block does not render the table due to https://github.com/quarto-dev/quarto-cli/issues/6134 (see https://arviz-devs.github.io/ArviZ.jl/previews/PR279/creating_custom_plots/)